test/e2e/framework/assertions.go

```diff
@@ -44,19 +46,19 @@ func WithPollInterval(d time.Duration) OptionFn {
 	}
 }
 
-// AssertResourceNeverExists asserts that a statefulset is never created for the duration of CustomForeverTestTimeout
+// AssertResourceNeverExists asserts that a statefulset is never created for the duration of DefaultTestTimeout
 func (f *Framework) AssertResourceNeverExists(name, namespace string, resource client.Object, fns ...OptionFn) func(t *testing.T) {
 	option := AssertOption{
 		PollInterval: 5 * time.Second,
-		WaitTimeout:  CustomForeverTestTimeout,
+		WaitTimeout:  DefaultTestTimeout,
 	}
 	for _, fn := range fns {
 		fn(&option)
 	}
 
 	return func(t *testing.T) {
 		//nolint
-		if err := wait.Poll(option.PollInterval, option.WaitTimeout, func() (done bool, err error) {
+		if err := wait.Poll(option.PollInterval, option.WaitTimeout, func() (bool, error) {
```

Comment:
Is there any reason on why the named parameters have been dropped? IMHO they help on code readability.

Reply:
before my change every wait.Poll was with no named parameters. Just to maintain the coherence.